src/devices/bus/spectrum/neogs.h

```diff
@@ -67,4 +60,6 @@ class neogs_device : public device_t
 	u8 m_vol[8];
 };
 
+DECLARE_DEVICE_TYPE(NEOGS, neogs_device)
```

Comment:
Please use a prefix on the device type to give a form of crude namespacing, e.g. call it `ZXBUS_NEOGS`.